lib/screen_controller.dart

```diff
@@ -147,11 +147,11 @@ class ScreenController {
         final res = await Navigator.push(
           context,
           MaterialPageRoute(builder: (context) => CameraScreen(
-            mode: action.options!['mode'] != null ? Utils.getString(action.options!['mode'], fallback: '') : 'DEFAULT',
-            initialCamera: action.options!['initialCamera'] != null ? Utils.getString(action.options!['initialCamera'], fallback: '') : 'DEFAULT',
-            useGallery: action.options!['useGallery'] != null ? Utils.getBool(action.options!['useGallery'], fallback: false) : true,
-            preview: action.options!['preview'] != null ? Utils.getBool(action.options!['preview'], fallback: false) : false,
-            maxCount: action.options!['maxCount'] != null ? Utils.getInt(action.options!['maxCount'], fallback: 0) : 1,
+            mode: action.options == null? 'DEFAULT' : action.options!['mode'] != null ? Utils.getString(action.options!['mode'], fallback: '') : 'DEFAULT',
```

Comment:
Don't use 'Default' as it is non-descriptive (doesn't tell you anything) and is another state to keep track of. Either use null (I prefer this for simple attribute) or a name (video_and_photo).

This applies for all the attributes also. Don't use the name `default`